source/deployment/scaling.rst

```diff
@@ -4,32 +4,34 @@ Scaling for Enterprise
 
 Mattermost is designed to scale from small teams hosted on a single server to large enterprises running in cluster-based, highly available deployment configurations. 
 
-- Server requirements vary based on usage and it is highly recommended that pilots are run before enterprise-wide deployments in order to estimate full scale usage based on your specific organizational needs. 
+Server requirements vary based on usage and it is highly recommended that pilots are run before enterprise-wide deployments in order to estimate full scale usage based on your specific organizational needs. 
 
 Single Machine Deployment 
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Organizations less than 500 users can typically run on a single server. 
+Organizations less than 2,000 users can typically run on a single server. 
```

Comment:
Is the number "2000" correct?

Reply:
It really depends on usage. Pre-release is a 4,500 user server and I would be very comfortable running it on a single 2 vCPU 4GB RAM machine.